tools/ovc/openvino/tools/ovc/moc_frontend/pytorch_frontend_utils.py

```diff
@@ -44,6 +68,19 @@ def extract_module_extensions(args):
                 "NNCF models produced by nncf<2.6 are not supported directly. Please upgrade nncf or export to ONNX first.")
     inputs = prepare_torch_inputs(example_inputs)
     if not isinstance(model, (TorchScriptPythonDecoder, TorchFXPythonDecoder)):
+        if isinstance(model, (str, pathlib.Path)):
+            # attempt to load scripted model
+            try:
+                model = torch.jit.load(model)
+            except:
+                pass
+        if hasattr(torch, "export") and isinstance(model, (str, pathlib.Path)):
+            # attempt to load exported model
+            try:
+                model = torch.export.load(model)
+            except:
+                pass
```

Comment:
Write the message to user if you couldn't load the model from .pt file with the message why that can happen, eg if user using checkpoint instead of the model

Reply:
It is not possible to determine model format by file extension. Even TF model can have such extension.
We deduce PT format by attempt to load it as `TorchScript` or `ExportedProgram`